src/Elastic.Apm.Profiler.IntegrationsGenerator/Elastic.Apm.Profiler.IntegrationsGenerator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>

```

Comment:
This is not a required change for this PR.
But again (similar to [here](https://github.com/elastic/apm-agent-dotnet/pull/1933/files#r1037308425)) I encountered random errors running this, so I simply upgraded.